# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/moz.build
# Commit: f16cf2573a4d
# Full Hash: f16cf2573a4dafa7d909ec43a81c5379480ca994
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Move AudioConduit to use upstream for audioLevel and remove unused code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102276
# ==============================================================================

diff -r 90a15d97399e -r f16cf2573a4d dom/media/webrtc/libwebrtcglue/moz.build
--- a/dom/media/webrtc/libwebrtcglue/moz.build	Mon Jan 18 11:07:55 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/moz.build	Mon Jan 18 11:16:22 2021 +0100
@@ -20,7 +20,6 @@
     "AudioConduit.cpp",
     "GmpVideoCodec.cpp",
     "MediaDataCodec.cpp",
-    "RtpSourceObserver.cpp",
     "VideoConduit.cpp",
     "VideoStreamFactory.cpp",
     "WebrtcGmpVideoCodec.cpp",